### PR TITLE
@craigspaeth => Team Channel QA

### PR DESCRIPTION
**Pull request by kanaabe**
### Info:

Closes https://github.com/artsy/force/issues/205

`channel_nav` is now a component (shared by team channel landing page + team channel article page).

### Screenshots:
image

image

image

### CODE REVIEW

File: analytics/articles.js [hunk 1]
```diff
@@ -41,12 +41,15 @@ if(location.pathname.match('/article/')){
 
   }).on('click', '.article-section-callout', function(){
     var articleId = $(this).closest('.article-container').data('id');
-    analytics.track('Clicked article impression', {
-      article_id: articleId,
-      destination_path: $(this)[0].href.replace(/^.*\/\/[^\/]+/, ''),
-      impression_type: 'article_callout',
-      context_type: 'article_fixed'
-    });
+    // Only track callouts that are links
```
Comment (craigspaeth) [inline analytics/articles.js]: 👍

File: analytics/article_impressions.js [hunk 2]
```diff
@@ -143,16 +143,17 @@ if(location.pathname.match('/article/') || location.pathname.match('/articles'))
               }
             }
           }
-        }else if(classList.contains('article-section-callout')){
+        }else if(classList.contains('article-section-callout') && $(this)[0].href){
+          var destination_path = $(this)[0].href.replace(/^.*\/\/[^\/]+/, '')
```
Comment (craigspaeth) [inline analytics/article_impressions.js]: camelCase pl0x `destinationPath`

File: apps/article/templates/article.jade [hunk 3]
```diff
@@ -8,7 +8,15 @@ append locals
   - bodyClass = bodyClass + ' ' + article.getBodyClass()
 
 block body
-  #articles-show
+
+  //- Render channel nav if article is in a team channel
+  - var showClass = ''
+  if sd.ARTICLE_CHANNEL && sd.ARTICLE_CHANNEL.isTeam()
```
Comment (craigspaeth) [inline apps/article/templates/article.jade]: This `.isTeam()` seems odd. `sd` should be left for the "pure" JSON data. In this case you probably want to do `articleChannel = new Channel sd.ARTICLE_CHANNEL` first.

File: components/channel_nav/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+The channel nav is currently used in two places: team channel landing pages, and team channel article pages.
```
Comment (craigspaeth) [inline components/channel_nav/README.md]: 🙌  docs!

File: analytics/article_impressions.js (hunk 2 above)
Comment (kanaabe) [inline analytics/article_impressions.js]: 👍

File: apps/article/templates/article.jade (hunk 3 above)
Comment (kanaabe) [inline apps/article/templates/article.jade]: ah yeah, that's much better

**Comment by craigspaeth** [comment]
Nice refactor! I have some minor comments, but nothing blocking. Feel free to merge when you're ready. 👍

**Comment by kanaabe** [comment]
Thanks for the comments! I addressed the points you mentioned. Mergin'

**Pull request merged**